### ISSUE TITLE
Is this library deprecated or not supported anymore? 

### ISSUE THREAD

**Issue by tonmanayo** [sogilis/react-native-bluetooth-manager#17]
The example in the repo doesn't compile

doesn't look like there is active work being done. 

Would really like to use this library, but I see it being an issue if it's not being updated or worked on anymore

**Comment by sdenier** [comment]
Indeed, support for this library will largely be opportunistic on our side. To this date we only have one project which uses this library, and it does not need active support. There are a few forks but none made a PR to enhance/fix issues.
